### ISSUE TITLE
[Loss functions] Compare MAE and MSE

### ISSUE THREAD

**Issue by kapiblue** [JankowskiDaniel/Neural-Deep-Retina#70]
Check what is the training stability and test set performance of models trained with MAE and MSE.

**Comment by kapiblue** [comment]
A comparison between baseline and CfC.

I trained each configuration just once. MAE could be better for baseline and MSE could be better for CfC.

Report: https://api.wandb.ai/links/jankowskidaniel06-put/4ysownkm
